Adding History Shard term (#1551)
temporalio/documentation: docs/concepts/what-is-a-history-service.md

---
id: what-is-a-history-service
title: What is a History Service?
description: The History Service is responsible for persisting Workflow Execution state and determining what to do next to progress the Workflow Execution through History Shards.
sidebar_label: History Service
tags:
  - term
---

> The History Service is responsible for persisting Workflow Execution state and determining what to do next to progress the Workflow Execution through [History Shards](/concepts/what-is-a-history-shard).

Reviewer: Would "...by using History Shards" be an accurate wording change?

> The History Service is responsible for persisting Workflow Execution state and determining what to do next to progress the Workflow Execution through [History Shards](/concepts/what-is-a-history-shard).

Author: ```suggestion
The History Service is responsible for persisting Workflow Execution state and determining what to do next to progress the Workflow Execution by using [History Shards](/concepts/what-is-a-history-shard).
```

![History Service](/diagrams/temporal-history-service.svg)

The total number of History Services can be between 1 and the total number of History Shards.
An individual History Service can support a large number of History Shards.

> Temporal recommends starting at a ratio of 1 History Service for every 500 History Shards.

Reviewer 2: this seems like duplicate of what is said in clusters.md

> The total number of History Services may increase between 1 and the total number of History Shards.
> Additionally an individual History Service can support a large number of History Shards.
> Temporal recommends starting at a ratio of 1 History Service for each 500 History Shards.

Reviewer: ```suggestion
The total number of History Services can be between 1 and the total number of History Shards.
An individual History Service can support a large number of History Shards.
Temporal recommends starting at a ratio of 1 History Service for every 500 History Shards.
```

> While the total number of History Shards remains static for the life of the Cluster the number of History Services can change.

Reviewer: ```suggestion
Although the total number of History Shards remains static for the life of the Cluster, the number of History Services can change.
```
